web-ts/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "isolatedModules": true,
     "jsx": "react",
     "lib": ["dom", "dom.iterable", "ES5", "ES6"],
+    "module": "ES2020",
```

Comment:
This could be the reason why sentry is no longer working as sentry is loaded in a deferred way.